tests/unit/states/test_ldap.py

```diff
@@ -8,13 +8,17 @@
 with something sensible.
 """
 import copy
+import logging
 
 import salt.states.ldap
 from salt.utils.oset import OrderedSet
 from salt.utils.stringutils import to_bytes
 from tests.support.mixins import LoaderModuleMockMixin
 from tests.support.unit import TestCase
 
+log = logging.getLogger(__name__)
```

Comment:
I'm not seeing log being used anywhere.

Also, can these tests be converted to pytest?